src/styles/catalog.scss

```diff
@@ -0,0 +1,9 @@
+.catalog {
+  margin: 0;
```

Comment:
BEM couldn't have any margins, if you want to apply them use "mix" or just reset the default browser styles with reset/normillize.